### PR TITLE
NEXT-10072 - Plugin compatibility

### PR DESCRIPTION
**Pull request by DennisGarding**

### CODE REVIEW

File: tests/UsedClassesAvailableTest.php [hunk 1]
```diff
@@ -12,25 +12,16 @@ class UsedClassesAvailableTest extends TestCase
...
-            $classRelativePath = str_replace(['.php', '/'], ['', '\\'], $class->getRelativePathname());
-
-            $this->getMockBuilder($namespace . '\\' . $classRelativePath)
-                ->disableOriginalConstructor()
-                ->getMock();
-        }
-
-        // Nothing broke so far, classes seem to be instantiable
-        $this->assertTrue(true);
+        static::assertCount(6, $files);
```
Comment (mitelg) [inline tests/UsedClassesAvailableTest.php]: now you only check, if the classes are present. before there was a check if they were instantiatable

**Review by mitelg — CHANGES_REQUESTED** [review]

File: tests/UsedClassesAvailableTest.php (hunk 1 above)
Comment (DennisGarding) [reply tests/UsedClassesAvailableTest.php]: there are no instantiatable php classes

**Comment by mitelg** [comment]
maybe we could add a small e2e test to check if the tab is present in the admin

**Review by mitelg — CHANGES_REQUESTED** [review]
same comments as here https://github.com/shopware/swag-docs-custom-cms-block/pull/6

File: Resources/app/Administration/test/e2e/cypress/support/commands.js [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+// -- This is will overwrite an existing command --
+// Cypress.Commands.overwrite("visit", (originalFn, url, options) => { ... })
+
+/**
+ * Switches administration UI locale to EN_GB
+ * @memberOf Cypress.Chainable#
+ * @name setLocaleToEnGb
+ * @function
+ */
+Cypress.Commands.add('setLocaleToEnGb', () => {
```
Comment (mitelg) [inline Resources/app/Administration/test/e2e/cypress/support/commands.js]: check commands

**Review by mitelg — CHANGES_REQUESTED** [review]

**Review by mitelg — APPROVED** [review]

**Pull request merged**